### PR TITLE
[15.0][MIG] account_invoice_inter_company

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration
- Fixed pre-commit issues
- Fixed test with correct account chart loading without demo
- Changing view issues for new res_config view structure
@ForgeFlow

### CODE REVIEW

**Comment by etobella** [comment]
@ChrisOForgeFlow Can you rebase it?

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow Can you rebase it?

Done!

**Review by AaronHForgeFlow — COMMENTED** [review]
Overall looks good so far.

I have trouble to validate an invoice when the sales taxes are different percentage from the vendor taxes. I cannot validate a vendor bill that was created from a intercompany sales order:

> This is an autogenerated multi company invoice and you're trying to modify the amount, which will differ from the source one

I guess this was done on purpose but I don't think this is correct. Should it just count the untaxed amount?

**Comment by AaronHForgeFlow** [comment]
Hi @ChrisOForgeFlow , can you have a look to my previous comment? Thanks in advance.

**Comment by ChrisOForgeFlow** [comment]
> Overall looks good so far.
> 
> I have trouble to validate an invoice when the sales taxes are different percentage from the vendor taxes. I cannot validate a vendor bill that was created from a intercompany sales order:
> 
> > This is an autogenerated multi company invoice and you're trying to modify the amount, which will differ from the source one
> 
> I guess this was done on purpose but I don't think this is correct. Should it just count the untaxed amount?

I agree, the taxes can differ between the sale and the purchase, I have added the change so that you can compare between the amounts without taxes

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Review by mariadforgeflow — APPROVED** [review]

File: account_invoice_inter_company/__manifest__.py [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
     "name": "Inter Company Invoices",
     "summary": "Intercompany invoice rules",
-    "version": "14.0.1.0.3",
+    "version": "15.0.1.0.0",
     "category": "Accounting & Finance",
     "website": "https://github.com/OCA/multi-company",
     "author": "Odoo SA, Akretion, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "depends": ["account"],
+    "depends": ["account", "base_setup"],
```
Comment (yankinmax) [inline account_invoice_inter_company/__manifest__.py:15]: we don't need here `base_setup` as `account` already depends on it

**Review by yankinmax — APPROVED** [review]

**Review by etobella — APPROVED** [review]
Can you apply the change from @yankinmax

**Review by antoniocanovas — APPROVED** [review]
Functional review OK

**Pull request merged**